### DISCUSSION TITLE
希望增加一个独立窗口的功能

### DISCUSSION THREAD

**Discussion by fo2w** [he3-app/he3-feedback#93] [Ideas]
### 场景举例:

一段JSON密文, 我先用AES解密, 然后用JSON格式化工具格式化

在现有的交互中, 我需要不断在这两个功能间切换, 且每次切换回AES解密后, 原先配置的密钥等都要重新配置一遍

### 期望的一种实现:

有一个按钮可以将当前工具独立为一个新窗口, 可以置顶.

该新窗口独立工作, 与主窗口互不影响.

### 改进后的交互:

JSON工具独立为新窗口, 并置顶, 主要视觉在主窗口处理加解密相关事宜

需要格式化的时候复制并粘贴到独立的JSON格式化窗口完成相关操作

**Comment by hongchaodeng** [comment]
你好，我们正在规划相关的数据历史记录功能。之后每个输出数据都能够被保存，并且直接右键打开某个工具作为输入。敬请期待！

**Comment by zhenweiwang1990** [comment]
工具单独打开窗口功能也在规划中了，感谢建议反馈。

**Comment by fo2w** [comment]
收到, 感谢付出.
